app/src/main/java/chat/rocket/android/authentication/ui/AuthenticationActivity.kt

```diff
@@ -23,18 +24,28 @@ class AuthenticationActivity : AppCompatActivity(), HasSupportFragmentInjector {
     @Inject lateinit var fragmentDispatchingAndroidInjector: DispatchingAndroidInjector<Fragment>
     @Inject lateinit var presenter: AuthenticationPresenter
     val job = Job()
+    val PREF_FIRST_TIME_USER = "first_time_user"
 
     override fun onCreate(savedInstanceState: Bundle?) {
         AndroidInjection.inject(this)
         setContentView(R.layout.activity_authentication)
         setTheme(R.style.AuthenticationTheme)
         super.onCreate(savedInstanceState)
 
+        var isFirstTimeUser = Utils().readSharedSetting(this,PREF_FIRST_TIME_USER, "true")!!.toBoolean()
```

Comment:
Take a look at https://github.com/RocketChat/Rocket.Chat.Android/blob/develop/app/src/main/java/chat/rocket/android/infrastructure/LocalRepository.kt for saving this information and how we use it to check for a first time on https://github.com/RocketChat/Rocket.Chat.Android/blob/develop/app/src/main/java/chat/rocket/android/app/RocketChatApplication.kt#L102

Reply:
@luciofm can you tell me how to check for first time user. The code has been updated.